templates/patterns/carousel/carousel.ui_patterns.yml

```diff
@@ -5,22 +5,22 @@ carousel:
     items:
       type: "array"
       label: "Carousel items"
-      description: "List of items compatible with the Page banner structure."
+      description: "List of items compatible with the ECL Banner component structure."
       preview:
         - title: "Lorem ipsum dolor sit amet"
           description: "Nullam sollicitudin suscipit diam, ac blandit ipsum tempor consectetur"
           url: "https://example.com"
           url_text: "CTA"
           image: "https://placeimg.com/640/480/tech"
-          variant: "image"
+          variant: "text-box"
           centered: false
           credit: "© Copyright first item"
         - title: "Duis vitae pulvinar turpis"
           description: "Integer quis lorem tellus. Nullam sollicitudin suscipit diam, ac blandit ipsum tempor consectetur"
           url: "https://example.com"
           url_text: "CTA"
           image: "https://placeimg.com/640/480/arch"
-          variant: "image-gradient"
+          variant: "text-highlight"
```

Comment:
It can be a tricky moment for other non-EWPP sites if they will try to use not existing anymore variants.
Does it make sense somehow deprecate variants (and in a nutshell show different variant)?